lib/features/vpn/vpn_switch.dart

```diff
@@ -49,42 +51,68 @@ class _VPNSwitchState extends State<VPNSwitch> {
         });
       });
     } else {
-      // This ui for desktop
-      return AdvancedSwitch(
```

Comment:
I had to remove AdvancedSwitch and use new one, reason being there was issue with control.

Reply:
What issue are you experiencing with the control..? It works fine as is for me:

https://github.com/user-attachments/assets/88cafe37-2473-47f3-849f-bb4d84b71dbf

Reply:
I definitely like the change to use animated_toggle_switch now so I think the control issues are irrelevant.